tox.ini

```diff
@@ -0,0 +1,8 @@
+[tox]
+envlist = py35,py36,py38,py39
```

Comment:
I have noticed the missing dependency for python3.7. Also this should not work for python 3.5 (f-strings usage)

Reply:
should be: `py36,py37,py38,py39`.